sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -530,8 +533,8 @@ def plot_confusion_matrix(estimator, X, y_true, *, labels=None,
     >>> clf = SVC(random_state=0)
     >>> clf.fit(X_train, y_train)
     SVC(random_state=0)
-    >>> plot_confusion_matrix(clf, X_test, y_test)  # doctest: +SKIP
-    >>> plt.show()  # doctest: +SKIP
+    >>> plot_confusion_matrix(clf, X_test, y_test) # doctest: +SKIP
```

Comment:
We can add the missing space.

```suggestion
    >>> plot_confusion_matrix(clf, X_test, y_test)  # doctest: +SKIP
```

I agree that this is the best strategy to do here. We will be removing this functionality soon.